Modules/Filtering/LabelMap/include/itkLabelMap.h

```diff
@@ -267,7 +267,10 @@ class ITK_TEMPLATE_EXPORT LabelMap : public ImageBase<TLabelObject::ImageDimensi
    * Return the number of label objects in the image
    */
   typename Self::SizeValueType
-  GetNumberOfLabelObjects() const;
+  GetNumberOfLabelObjects() const
+  {
+    return static_cast<typename Self::SizeValueType>(m_LabelObjectContainer.size());
```

Comment:
Wouldn't `static_cast<SizeValueType>` just work here, inside the member function body?

Reply:
Yes, updated